_data/changelogs/component-button-group.yml

```
@@ -2,6 +2,21 @@ title: Button group
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Button group improved support for wrapping text.
+    summaryAdditional:  Improves support to button groups with wrapping lines of text.
```

Something like this? 
```suggestion
    summary: Improved the appearance of button groups when text wraps to multiple lines.
    summaryAdditional:  Now, every button in the group will be the same height. 
```